docker/Dockerfile

```diff
@@ -73,14 +73,14 @@ RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 RUN rm -rf tmp && git clone --depth 1 --branch ${LIGHTNING_VERSION} https://github.com/PennyLaneAI/pennylane-lightning.git tmp\
     && mv tmp/* /opt/pennylane-lightning && rm -rf tmp
-RUN pip install --no-cache-dir cmake ninja pybind11 wheel
+RUN pip install --no-cache-dir build cmake ninja pybind11 toml wheel
 
 # Download Lightning release and build lightning-qubit backend
 FROM base-build-python AS build-wheel-lightning-qubit
 WORKDIR /opt/pennylane-lightning
 RUN pip uninstall -y pennylane-lightning
-RUN python scripts/configure_pyproject_toml.py
-RUN python -m build
+RUN python scripts/configure_pyproject_toml.py || true
+RUN python -m build --wheel
```

Comment:
So, for my own knowledge, to have backward compatibility using the build module it is sufficient to add the `--wheel` flag, right?

Reply:
Yes.

Reply:
Beautiful!